### PR TITLE
Fix Gloo Mesh Gateway ci

### PR DESCRIPTION
**Pull request by kdorosh**
updates the setup script to DRY the 1.8+ istio operator versions, improves clarity of variable names, and fixes misleading documentation, and fixes a small bug with the discovered service mesh label selector (the last bug is the only one here that was required to fix ci in enterprise)

### CODE REVIEW

File: pkg/mesh-discovery/translation/mesh/detector/istio/istio_mesh_detector.go [hunk 1]
```diff
@@ -198,7 +198,7 @@ func getIngressGateway(
 	gatewayInfo := &discoveryv1.MeshSpec_Istio_IngressGatewayInfo{
 		Namespace:      svc.Namespace,
 		Name:           svc.Name,
-		WorkloadLabels: workloadLabels,
+		WorkloadLabels: svc.Spec.Selector,
```
Comment (EItanya) [inline pkg/mesh-discovery/translation/mesh/detector/istio/istio_mesh_detector.go:201]: Workload Labels is a really bad name, it should probably be selector labels or something, maybe we can document?

Comment (harveyxia) [reply pkg/mesh-discovery/translation/mesh/detector/istio/istio_mesh_detector.go:201]: this is a deprecated method of ingress gateway selection anyways

Comment (kdorosh) [reply pkg/mesh-discovery/translation/mesh/detector/istio/istio_mesh_detector.go:201]: https://github.com/solo-io/gloo-mesh/pull/1726/commits/2b6115ee1963f6b86ae9f74f9ba5c770cc09fa13

File: ci/setup-funcs.sh [hunk 2]
```diff
@@ -315,7 +315,8 @@ EOF
 # Operator spec for istio 1.7.x
 function install_istio_1_7() {
   cluster=$1
-  ingressPort=$2
+  eastWestIngressPort=$2
```
Comment (npolshakova) [inline ci/setup-funcs.sh:318]: 🎉

**Review by npolshakova — APPROVED** [review]

**Pull request merged**